docs/tutorials/fundamentals/tutorial-swapping-token.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Tutorial - Swapping UOS from Ultra to Ethereum'
+order: -99993
+outline: [0, 7]
+---
+
+# Tutorial - Swapping UOS from Ultra to Ethereum
+
+This tutorial guides you through the process of swapping UOS tokens from the Ultra network to UOS ERC-20 tokens on the Ethereum network using the Ultra Client. This process is designed for users who already have an Ethereum account and are familiar with cryptocurrency transactions.
+
+## Step 1: Prepare the Swap in the Ultra Client
+
+Start the swap process by opening the Ultra Client. Navigate to the Token Swap section.
+
+1. Enter the amount of UOS you want to swap. In this example, we will swap 3000 UOS.
+2. Enter your Ethereum address where you want to receive the UOS ERC-20 tokens. Make sure this is an address you control.
+
+![](./images/ultra-swap-ultra-client.png)
```

Comment:
Better highlight where you need to press to reach this screen like so, but in a better way
![image](https://github.com/ultraio/docs-blockchain/assets/27926086/f1afd54d-46a3-4041-b75c-d1184c78112b)